common/main/en_AU.xml

```diff
@@ -1617,7 +1617,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<dateFormats>
 					<dateFormatLength type="full">
 						<dateFormat>
-							<pattern>↑↑↑</pattern>
+							<pattern>EEEE d MMMM y G</pattern>
```

Comment:
According to the information I have, and is included in the ticket, this format should retain a comma after the weekday. It is only when the weekday OR the year is omitted that the comma goes away.

Reply:
> According to the information I have, and is included in the ticket, this format should retain a comma after the weekday. It is only when the weekday OR the year is omitted that the comma goes away.

@Fredrik Yes but per the Google linguists, the comma should also be omitted in that case if the weekday is fully spelled out. The examples from Apple only showed an abbreviated weekday.

Reply:
The feedback from Apple's vetters were abbreviation-agnostic I guess you could say. The example values in the ticket were added by Annemarie I believe.